var/spack/repos/builtin/packages/margo/package.py

```diff
@@ -7,43 +7,49 @@
 
 
 class Margo(AutotoolsPackage):
-    """Argobots bindings to the Mercury RPC API"""
+    """A library that provides Argobots bindings to the Mercury RPC
+    implementation.  This name will be deprecated soon; please use the
+    mochi-margo package instead."""
 
-    homepage = "https://xgitlab.cels.anl.gov/sds/margo"
-    url      = "https://xgitlab.cels.anl.gov/sds/margo/-/archive/v0.4.3/margo-v0.4.3.tar.gz"
-    git      = "https://xgitlab.cels.anl.gov/sds/margo.git"
+    homepage = 'https://xgitlab.cels.anl.gov/sds/margo'
+    git = 'https://xgitlab.cels.anl.gov/sds/margo.git'
 
-    maintainers = ['fbudin69500', 'chuckatkins', 'carns']
+    maintainers = ['carns', 'mdorier']
```

Comment:
Should we keep @fbudin69500 @chuckatkins as maintainers/add them to `mochi-margo`?